pkg/apis/autoscaling/register.go

```diff
@@ -67,13 +67,23 @@ const (
 	// RPS is the requests per second reaching the Pod.
 	RPS = "rps"
 
+	// MemorySuffixAnnotationKey is the annotation to specify which memory suffix
```

Comment:
This is totally a nit, but for readability would it be better to move this section after the target one? It was a little jarring to see the target used in an example without having seen its explanation yet.